seed/challenges/00-getting-started/getting-started.json

```diff
@@ -85,6 +85,81 @@
       "tests": [],
       "type": "Waypoint",
       "challengeType": 7,
+      "titleId": "Mengetahui Bagaimana Free Code Camp bekerja",
+      "descriptionId": [
+	[
+	  "//i.imgur.com/6ibIavQ.jpg",
+	  "Gambaran 4 kelebihan Free Code Camp: Terhubung, Belajar JavaScript, Membangun Portofolio, Membantu nirlaba",
```

Comment:
Wouldn't "nonprofits" be better translated to _organisasi nirlaba_? Can you use _nirlaba_ as a noun?